### PR TITLE
remove redundant modules in setup

### PR DESCRIPTION
**Pull request by andreas-theilgaard**
This is the minimal changes needed in order to only install `MSDeformAttn`. 

I have kept the rest of the repo as is, even though we only are using a fraction of it. 

I am not sure how the exact pipeline is going to be. But if just doing
`tao_pytorch_backend = {git = "https://github.com/recoord/tao_pytorch_backend", rev = "master"}`
Then, we assume we have already installed `torch` right? Because `setup.py` depends on `torch` as the only requirement. 
If we don't have `torch` installed at 'install' time then I guess we could add `install_requires=["torch==2.2.1"]` to `setup.py`.
But this way we need to ensure that the torch versions in sunstone and tao_pytorch_backend are in sync (even though we probably can get away installing MSDeformAttn using any newer torch version).

### CODE REVIEW

File: release/python/utils/utils.py [hunk 1]
```diff
@@ -139,6 +139,14 @@ def get_extra_compile_args():
         "-D__CUDA_NO_HALF_OPERATORS__",
         "-D__CUDA_NO_HALF_CONVERSIONS__",
         "-D__CUDA_NO_HALF2_OPERATORS__",
+        '-gencode=arch=compute_75,code=sm_75',  # Turing GPUs (T4)
```
Comment (andreas-theilgaard) [inline release/python/utils/utils.py:142]: Adding this enable us to just have one wheel. I.e no need to have a wheel for each compute capability version. We should have done this in the first place in sunstone. But I guess you learn something new every day

**Comment by tslott** [comment]
> I have kept the rest of the repo as is, even though we only are using a fraction of it.

I think that's fine for now. I can do a followup PR that cleans up the rest. Let's make this "install from repo"-approach work and then make it pretty :) 

> I am not sure how the exact pipeline is going to be. ............

Yes. Still investigating what is the best option, but it will probably be something like "build python wheel through github action in this repo, store as release, pip install the wheel in sunstone". But the idea was definitely to build the tao backend where we have torch available, and then install the resulting python wheel in sunstone.

File: release/python/utils/utils.py (hunk 1 above)
Comment (tslott) [reply release/python/utils/utils.py:142]: Cool. I'm pretty sure I saw some code somewhere, where they handle compiling to different architectures, but is this works, then coolio.

**Review by tslott — APPROVED** [review]
Thanks for the PR 💪 I will merge this PR to make testing a bit easier. The repo is not used for anything right now anyway.

**Pull request merged**